File: UNIDAD 5/JAVA/U5_P3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u5_p3;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class U5_P3 {

    public static void main(String[] args) {
        Scanner L = new Scanner(System.in);
        double x,y;
        System.out.println("Escribe el valor para x: ");
        x = L.nextDouble();
        System.out.println("Escribe el valor para y: ");
        y = L.nextDouble();
        System.out.println("Resultado ecuacion1: " + ecuacion1(x,y));
        System.out.println("Resultado ecuacion2: " + ecuacion2(x,y));
        
    }
    public static double ecuacion1(double x, double y){
        double r;
        r = Math.pow(x,y) + (2*Math.pow(y, 4)); 
        return r;
    }
    public static double ecuacion2(double x, double y){
        double r;
        r = Math.sqrt(Math.pow(x, 4) + 5*y);
        return r;
    }
}

File: UNIDAD 5/C#/U5_P3.cs
using System;

namespace U5_P3
{
    internal class U5_P3
    {
        static void Main(string[] args) { 
         Console.WriteLine("Escribe el valor para x: ");
        double x = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("Escribe el valor para y: ");
        double y = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("Resultado ecuacion1: " + Ecuacion1(x, y));
        Console.WriteLine("Resultado ecuacion2: " + Ecuacion2(x, y));
    }

    public static double Ecuacion1(double x, double y)
    {
        double r = Math.Pow(x, y) + (2 * Math.Pow(y, 4));
        return r;
    }

    public static double Ecuacion2(double x, double y)
    {
        double r = Math.Sqrt(Math.Pow(x, 4) + 5 * y);
        return r;
    }
}
}